### ISSUE TITLE
文本识别训练慢

### ISSUE THREAD

**Issue by pmspku** [PaddlePaddle/PaddleOCR#6859]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：ubuntu2.0
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：2.3

cpu 核数14，读数据线程设置是2，
日志：迭代10次的时间总是1分钟，其中batch_cost包含reader_cost,log_cost是每打印一次日志的时间，batch_cost_1为计算方式为：
            loss_start_time=time.time()
            stats = {k: v.numpy().mean() for k, v in loss.items()}
            stats['lr'] = lr
            train_stats.update(stats)
            train_batch_cost_loss += time.time() - loss_start_time
疑问：怎么提高训练速度，目前batch-size=512,不明白为什么每次都是1分钟，分布式里面有什么限制吗？

reader_cost: 35.24063 s, batch_cost: 36.87850 s,batch_cost_1: 17.51441 s, log_cost: 54.40936 s,samples: 5120
reader_cost: 0.23762 s, batch_cost: 1.89268 s,batch_cost_1: 53.27949 s, log_cost: 55.18292

**Comment by pmspku** [comment]
batch size 设置越大，反而越慢，当设置为128时，iter=10 耗时10s，而512时，iter=10 耗时53s,反而快一点，这是为什么呢，总感觉有个上限设置？

**Comment by Alice-Thirty** [comment]
me, too
batch size  72 is faster than  batch size  128
